init.sh

```diff
@@ -2,17 +2,18 @@ platform='unknown'
 sw_afs=0
 unamestr=`uname`
 
-export LCGPATH=/afs/cern.ch/sw/lcg/views/LCG_83/x86_64-slc6-gcc49-opt
-#if [ -z ${FCCPAPASCPP+x} ]; then
+if [ -z ${FCCPAPASCPP+x} ]; then
     unset $FCCPAPASCPP
     echo "FCCPAPASCPP is unset, setting to $PWD/install";
     export FCCPAPASCPP=$PWD/install
     export PATH=$FCCPAPASCPP/bin:$PATH    
-#fi
+fi
 
 if [[ "$unamestr" == 'Linux' ]]; then
     platform='Linux'
     export LD_LIBRARY_PATH=$FCCPAPASCPP/lib:$LD_LIBRARY_PATH
+    source /cvmfs/fcc.cern.ch/sw/0.8/init_fcc_stack.sh $1
```

Comment:
Hi Alice,

Can you update this to 0.8.1? That should also fix the problem that made the test fail.

Cheers,
Joschka